src/Legacy/DynamoRevitVersionSelector/VersionSelectorApp.cs

```diff
@@ -126,6 +149,15 @@ private SplitButton AddSplitButtonGroup(RibbonPanel panel)
             var versionData = new SplitButtonData("versions", Resources.DynamoVersions);
             var button = panel.AddItem(versionData) as SplitButton;
 
+            Bitmap dynamoIcon = Resources.dynamo_32x32;
+
+            BitmapSource bitmapSource =
+                Imaging.CreateBitmapSourceFromHBitmap(
+                    dynamoIcon.GetHbitmap(),
```

Comment:
You need to delete this object using `GDI DeleteObject` method to free the memory used by the bitmap object. You should use something like:
```
// at class level
[System.Runtime.InteropServices.DllImport("gdi32.dll")]
public static extern bool DeleteObject(IntPtr hObject);

// your code
using (Bitmap dynamoIcon = Resources.dynamo_32x32) 
{
    IntPtr hBitmap = bmp.GetHbitmap(); 

    try 
    {
        var bitmapSource = Imaging.CreateBitmapSourceFromHBitmap(hBitmap, IntPtr.Zero, Int32Rect.Empty, BitmapSizeOptions.FromEmptyOptions());
    }
    finally 
    {
        DeleteObject(hBitmap)
    }
}
```